circle-test.sh

```diff
@@ -8,7 +8,7 @@ GORACE="halt_on_error=1"
 BUILD_DIR=$HOME/influxdb-build
 GO_VERSION=go1.4
 PARALLELISM="-parallel 256"
-TIMEOUT="-timeout 300s"
+TIMEOUT="-timeout 420s"
```

Comment:
I got a timeout at first so had to raise it.  I'm seeing #2343 now too w/ the partial test.